src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsResponseDetails.java

```diff
@@ -13,6 +19,93 @@ public FeedbackRankRecipientsResponseDetails() {
         answer = Const.POINTS_NOT_SUBMITTED;
     }
 
+    /**
+     * Provides updates of responses for 'rank recipient question', such that the ranks in the responses are consistent.
+     * @param responses responses to one feedback question, from one giver
+     * @param maxRank the maximum rank in each response
+     * @return a list of {@code UpdateOptions} that contains the updates for the responses, if any
+     */
+    public static List<FeedbackResponseAttributes.UpdateOptions> makeConsistencyUpdateForOneQuestionFromOneGiver(
```

Comment:
Since the function returns a list of `UpdateOptions` for "rank recipient" questions, perhaps the name can be something that is more descriptive of this use, e.g. `getUpdateOptionsForRankRecipientQuestions`?